Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/DriverInformation.hs

```diff
@@ -208,22 +210,55 @@ findAllWithLimitOffsetByMerchantId mbSearchString mbSearchStrDBHash mbLimit mbOf
 
 getDriversWithOutdatedLocationsToMakeInactive :: Transactionable m => UTCTime -> m [Person]
 getDriversWithOutdatedLocationsToMakeInactive before = do
-  findAll $ do
-    (driverInformation :& _ :& person) <-
-      from $
-        table @DriverInformationT
-          `innerJoin` table @DriverLocationT
-            `Esq.on` ( \(driverInformation :& drLoc) ->
-                         driverInformation ^. DriverInformationDriverId ==. drLoc ^. DriverLocationDriverId
-                           &&. drLoc ^. DriverLocationUpdatedAt <. val before
-                     )
-          `innerJoin` table @PersonT
-            `Esq.on` ( \(driverInformation :& _ :& person) ->
-                         driverInformation ^. DriverInformationDriverId ==. person ^. PersonTId
-                     )
-    where_ $ driverInformation ^. DriverInformationActive
-    orderBy [asc $ driverInformation ^. DriverInformationUpdatedAt]
-    pure person
+  driverInfos <- getDriverInfos
+  driverLocations <- getDriverLocs driverInfos before
```

Comment:
Can you fetch `driverLocations` first? This will filter out all the irrelevant rows first.

`driverInformation` will give a larger set of entries

Reply:
Sure 👍